static/app/components/editableText.tsx

```diff
@@ -132,121 +131,59 @@ function EditableText({
   }
 
   return (
-    <Wrapper>
-      <InnerWrapper ref={innerWrapperRef} isDisabled={isDisabled} isEditing={isEditing}>
-        {isEditing ? (
-          <InputWrapper isEmpty={isEmpty} data-test-id="editable-text-input">
-            <StyledField inline={false} flexibleControlStateSize stacked>
-              <StyledInput
-                name={name}
-                ref={inputRef}
-                value={inputValue}
-                onChange={handleInputChange}
-              />
-            </StyledField>
-            <InputLabel>{inputValue}</InputLabel>
-          </InputWrapper>
-        ) : (
-          <React.Fragment>
-            <Label
-              onClick={isDisabled ? undefined : handleEditClick}
-              ref={labelRef}
-              data-test-id="editable-text-label"
-            >
-              <InnerLabel>{inputValue}</InnerLabel>
-            </Label>
-            {!isDisabled && <StyledIconEdit />}
-          </React.Fragment>
-        )}
-      </InnerWrapper>
+    <Wrapper ref={innerWrapperRef} isDisabled={isDisabled} isEditing={isEditing}>
+      {isEditing ? (
+        <InputWrapper isEmpty={isEmpty} data-test-id="editable-text-input">
+          <StyledInput
+            name={name}
+            ref={inputRef}
+            value={inputValue}
+            onChange={handleInputChange}
+          />
+          <InputLabel>{inputValue}</InputLabel>
+        </InputWrapper>
+      ) : (
+        <React.Fragment>
+          <Label
+            onClick={isDisabled ? undefined : handleEditClick}
+            ref={labelRef}
+            data-test-id="editable-text-label"
+          >
+            <InnerLabel>{inputValue}</InnerLabel>
+            {!isDisabled && <IconEdit />}
+          </Label>
+        </React.Fragment>
+      )}
     </Wrapper>
   );
 }
 
 export default EditableText;
 
 const Label = styled('div')`
-  display: inline-block;
-  border-radius: ${p => p.theme.borderRadius};
-  text-align: left;
-  padding-left: 10px;
-  height: 40px;
```

Comment:
shouldn't the field have a height of 40px to be consistent with other fields that we currently have?

Reply:
It's not necessary since it's already getting set in the wrapper

Reply:
I can't not remember now but I think this might be the cause of the second gif issue